returnn/config.py

```diff
@@ -131,6 +131,58 @@ def get(self, key, default=None):
             return self.post_config[key]
         return self.config.get(key, default)
 
+    def update(self, other):
+        """
+        updates a ReturnnConfig with an other ReturnnConfig
+
+        :param ReturnnConfig other:
+        """
+        self.config.update(other.config)
+        self.post_config.update(other.post_config)
+        if other.staged_network_dict is not None:
+            self.staged_network_dict = other.staged_network_dict
+
+        if self.python_prolog is None:
+            self.python_prolog = other.python_prolog
+        elif isinstance(self.python_prolog, "str") and isinstance(
+            other.python_prolog, "str"
+        ):
+            self.python_prolog += other.python_prolog
```

Comment:
maybe something like this? so that we dont end up with `import numpy as npimport tensorflow as tf` in the `returnn.config`. or is this not an issue?

```suggestion
            self.python_prolog += "\n"
            self.python_prolog += other.python_prolog
```

Reply:
mh.. I think you are right. Also using newline here would be compatible to https://github.com/rwth-i6/i6_core/blob/48ec2dfd3d2dde9305892428f16c91f585d4aea8/returnn/config.py#L239-L240

lets do 
```suggestion
            self.python_prolog += "\n" + other.python_prolog
```

Reply:
yes thats cleaner